pyproject.toml

```diff
@@ -85,6 +82,11 @@ line_length = 120
 
 [tool.pylint]
 max-line-length = 120
+disable = [
```

Comment:
You can also write `too-few-public-methods` and `too-many-arguments`, instead of the error codes.

Nevertheless, I'm not sure we should do this globally... can you explain why you'd like to do that? And maybe add a short command behind the exclusions.